src/Microsoft.Tye.Hosting/Model/Application.cs

```diff
@@ -30,7 +30,7 @@ public void PopulateEnvironment(Service service, Action<string, string> set, str
                 // Inject normal configuration
                 foreach (var pair in service.Description.Configuration)
                 {
-                    set(pair.Name, pair.Value);
+                    set(pair.Name ?? "", pair.Value ?? "");
```

Comment:
Is this right? is a empty configuration value valid? or should we skip entries with null?

Reply:
Yeah I'll skip it, I don't think these will be null but might as well prevent it.